lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -49,24 +49,26 @@ const INITIAL_LTI_LAUNCH_STATE = {
  */
 export default function BasicLtiLaunchApp() {
   const {
-    authToken,
+    api: {
+      authToken,
+      // API callback to use to fetch the URL to show in the iframe. This is
+      // needed if resolving the content URL involves potentially slow calls
+      // to third party APIs (eg. the LMS's file storage).
+      viaCallbackUrl,
+    },
     authUrl,
     grading,
     lmsGrader,
     submissionParams,
     urls: {
       // Content URL to show in the iframe.
       via_url: viaUrl,
-      // API callback to use to fetch the URL to show in the iframe. This is
-      // needed if resolving the content URL involves potentially slow calls
-      // to third party APIs (eg. the LMS's file storage).
-      via_url_callback: viaUrlCallback,
```

Comment:
If PR #1 is merged, this will leave an empty dict here. So this will likely need to be fixed when that is merged before this one can go in (as both remove different halves). This is true in a few places in the python too, but this is the only place I could put a comment about it.